tests/mock_tests/fdborch/flush_syncd_notif_ut.cpp

```diff
@@ -0,0 +1,381 @@
+#include "../ut_helper.h"
+#include "../mock_orchagent_main.h"
+#include "../mock_table.h"
+#include "port.h"
+#define private public // Need to modify internal cache
+#include "portsorch.h"
+#include "fdborch.h"
+#include "crmorch.h"
+#undef private
+
+#define ETH0 "Ethernet0"
+#define VLAN40 "Vlan40"
+
+extern redisReply *mockReply;
+extern CrmOrch*  gCrmOrch;
+
+/*
+Test Fixture 
+*/
+namespace fdb_syncd_flush_test
+{
+    struct FdbOrchTest : public ::testing::Test
+    {   
+        std::shared_ptr<swss::DBConnector> m_config_db;
+        std::shared_ptr<swss::DBConnector> m_app_db;
+        std::shared_ptr<swss::DBConnector> m_state_db;
+        std::shared_ptr<swss::DBConnector> m_asic_db;
+        std::shared_ptr<swss::DBConnector> m_chassis_app_db;
+        std::shared_ptr<PortsOrch> m_portsOrch;
+        std::shared_ptr<FdbOrch> m_fdborch;
+
+        virtual void SetUp() override
+        {   
+
+            testing_db::reset();
+
+            map<string, string> profile = {
+                { "SAI_VS_SWITCH_TYPE", "SAI_VS_SWITCH_TYPE_BCM56850" },
+                { "KV_DEVICE_MAC_ADDRESS", "20:03:04:05:06:00" }
+            };
+
+            ut_helper::initSaiApi(profile);
+            
+            /* Create Switch */
+            sai_attribute_t attr;
+            attr.id = SAI_SWITCH_ATTR_INIT_SWITCH;
+            attr.value.booldata = true;
+            auto status = sai_switch_api->create_switch(&gSwitchId, 1, &attr);
+            ASSERT_EQ(status, SAI_STATUS_SUCCESS);
+
+            m_config_db = std::make_shared<swss::DBConnector>("CONFIG_DB", 0);
+            m_app_db = std::make_shared<swss::DBConnector>("APPL_DB", 0);
+            m_state_db = make_shared<swss::DBConnector>("STATE_DB", 0);
+            m_asic_db = std::make_shared<swss::DBConnector>("ASIC_DB", 0);
+
+            // Construct dependencies
+            // 1) Portsorch
+            const int portsorch_base_pri = 40;
+
+            vector<table_name_with_pri_t> ports_tables = {
+                { APP_PORT_TABLE_NAME, portsorch_base_pri + 5 },
+                { APP_VLAN_TABLE_NAME, portsorch_base_pri + 2 },
+                { APP_VLAN_MEMBER_TABLE_NAME, portsorch_base_pri },
+                { APP_LAG_TABLE_NAME, portsorch_base_pri + 4 },
+                { APP_LAG_MEMBER_TABLE_NAME, portsorch_base_pri }
+            };
+
+            m_portsOrch = std::make_shared<PortsOrch>(m_app_db.get(), m_state_db.get(), ports_tables, m_chassis_app_db.get());
+
+            // 2) Crmorch
+            ASSERT_EQ(gCrmOrch, nullptr);
```

Comment:
Why do we need an assert here?

Reply:
Not very important, but kinda notifies if any previous tests did not clear/free this